docs/decisions/README.md

```diff
@@ -0,0 +1,22 @@
+# Markdown Any Decision Records
```

Comment:
Could we add explicit role definitions (i.e., decider, consultant, informed)?

Reply:
The template files contains this description